[RHACS] ROX-14958: Adding steps to Configure proxy for on prem Secured Cluster services (#60706)
openshift/openshift-docs: modules/specifying-the-environment-variables-in-the-securedcluster-cr.adoc

// Module included in the following assemblies:
//
// * cloud_service/installing_cloud_ocp/configuring-the-proxy-for-collector-in-rhacs-cloud-service.adoc
:_content-type: PROCEDURE
[id="specifying-the-environment-variables-in-the-securedcluster-cr_{context}"]
= Specifying the environment variables in the SecuredCluster CR

[role="_abstract"]

> To configure an egress proxy, you can either use the cluster-wide {osp} proxy or specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables within the SecuredCluster Custom Resource (SecuredCluster CR) configuration file to ensure proper use of the proxy and bypass for internal requests within the specified domain.

Reviewer: ```suggestion
To configure an egress proxy, you can either use the cluster-wide {osp} proxy or specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables within the SecuredCluster Custom Resource (SecuredCluster CR) configuration file to ensure proper use of the proxy and bypass for internal requests within the specified domain.
The proxy configuration is applied to all running services, Sensor, Collector, Admission Controller and Scanner.
```

// Module included in the following assemblies:
//
// * cloud_service/installing_cloud_ocp/configuring-the-proxy-for-collector-in-rhacs-cloud-service.adoc
:_content-type: PROCEDURE
[id="specifying-the-environment-variables-in-the-securedcluster-cr_{context}"]
= Specifying the environment variables in the SecuredCluster CR

[role="_abstract"]

> To configure an egress proxy, you can either use the cluster-wide {osp} proxy or specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables within the SecuredCluster Custom Resource (SecuredCluster CR) configuration file to ensure proper use of the proxy and bypass for internal requests within the specified domain.

Reviewer 2: I think you can just have `CR` bc it the only thing that needs the abbreviation is CR 

```suggestion
To configure an egress proxy, you can either use the cluster-wide {osp} proxy or specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables within the SecuredCluster Custom Resource (CR) configuration file to ensure proper use of the proxy and bypass for internal requests within the specified domain.
```

> The proxy configuration is applied to all running services, Sensor, Collector, Admission Controller and Scanner.

Reviewer 2: ```suggestion
The proxy configuration applies to all running services, Sensor, Collector, Admission Controller and Scanner.
```
Mainly to avoid passive voice

> The proxy configuration is applied to all running services, Sensor, Collector, Admission Controller and Scanner.

Reviewer 2: I might be reading this sentence wrong. 

Are the the running services the `Sensor, Collector, Admission Controller and Scanner`? Because if so, I think there should be a colon after `all running services`.
Ex: 

```suggestion
The proxy configuration is applied to all running services: Sensor, Collector, Admission Controller and Scanner.
```

Author: @SimonBaeumer WDYT, can we go ahead with this suggestion?

Reviewer: +1

.Procedure

* Specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables under the customize specification in the SecuredCluster CR configuration file:
+
.Specifying environment variables

> ====

Reviewer 2: I don't think the code block needs to also be in a admonition block. You can just use the 
```
----
code 
----
```
code block format

> ====

Reviewer 2: Also need a `[source,yaml]` before the code block
